Tasks/AzureTestPlanV0/Invokers/goinvoker.ts

```diff
@@ -1,15 +1,34 @@
 import tl = require('azure-pipelines-task-lib/task');
 import utils = require('../utils');
 import constants = require('../constants');
+import tr = require("azure-pipelines-task-lib/toolrunner");
 
+//GO command like "gotestsum --junitfile TEST-Go<i>-junit.xml -- <filepath> -v -run ^<TestName>$"
 export async function executeGoTests(testsToBeExecuted: string[]): Promise<number> {
+    
+    let goPath = tl.which("go", true);
+    let go: tr.ToolRunner = tl.tool(goPath);
+    let argument = constants.INSTALL_GOTESTSUM;
+    go.line(argument);
+    await go.exec(<tr.IExecOptions>{ cwd: "" });
```

Comment:
exec should go to file https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/AzureTestPlanV0/testLibExecutor.ts

Reply:
I will update it.

Reply:
Updated.